auth/auth.go

```diff
@@ -179,7 +187,13 @@ func (auth *Auth) Configure(config core.ServerConfig) error {
 }
 
 func (auth *Auth) SupportedDIDMethods() []string {
-	return auth.supportedDIDMethods
+	// DID methods that don't require additional resources/configuration in the Nuts node are always supported.
+	// Other DID methods (did:nuts), are only supported if explicitly enabled.
+	result := []string{didweb.MethodName, didjwk.MethodName, didkey.MethodName, didx509.MethodName}
```

Comment:
did:web can also be disabled

Reply:
Thinking out loud: what would be the use case for NOT accepting did:web credentials?

- did:web is arguably unsafer compared to jwk, key, x509 (since these aren't suspectible to network-bound attacks, e.g. DNS or domain hijacking). So verifiers might not want to accept it.
- On the other hand, parties might want to accept it but might not want their Nuts node to create did:web DIDs.

Both are edge cases i.m.o., but I think the first argument is more credible. What do you think?